packages/clay-multi-select/src/index.tsx

```diff
@@ -77,6 +99,11 @@ export interface IProps extends React.HTMLAttributes<HTMLInputElement> {
 	 */
 	items: Array<Item>;
 
+	/**
+	 * Sets the name of the field to map the value/label of the item
+	 */
+	locator?: Locator;
```

Comment:
`locator` is something important that we should adopt not all multi selects adopt the same signature of `label, value` this may be different for each team as it will impact not only the data coming from the backend but also the data that goes to the backend.

Reply:
I think we used to have something similar in multi-select when it was only high-level. I think it was called "identifier", which defaulted to "id"

Reply:
Hmm, do you think it's better to call it `locator` or `identifier`? I think I'm fine with both of them even though I lean towards `locator` 😁.

Reply:
locator is fine with me. i would have leaned toward identifier if it was how to decipher the uniqueness of the item. like `classPK` or something. But since this is for both label and value, I think locator is fine.